bindings/C/c/adios2_c_internal.h

```diff
@@ -24,6 +24,8 @@ namespace
 template <int adios2_type>
 struct MapAdios2Type;
 
+inline adios2_error String2CAPI(const std::string &s, char *buf, size_t *size);
```

Comment:
not sure inline means anything. In any case, see comment about moving this function inside helper.

Reply:
Well, here it may not matter, but I'm pretty sure without the `inline` where the function is defined one will (sometimes) get an undefined symbol. Nevertheless, I wouldn't mind the function being in a .cpp file, but I didn't really want to add yet another file just for it.